packages/plugin-expo-app/app.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
       }
     }
 
+    client.addOnSession(session => {
+      if (Constants.manifest.revisionId) {
```

Comment:
Whats the reason for some of this being done here and others in the expo notifier setup?

Reply:
`appVersion` got moved to the Expo notifier setup because it must be set in configuration and therefore supplied before the notifier starts.

`codeBundleId` is not a configuration option and since there is nowhere to put it on the client, it needs to be added to session/event payloads in callbacks.